CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Change Log
 
 ## [Unreleased]
-- Added "observer" attribute to @property annotation
-- Added "computed" attribute to @property annotation
-- Fix bug where use of `Polymer.mixinBehaviors` with `@customElement` would throw an exception relating to the definition of the element's "properties" property.
+
+## [0.1.2] - 2018-01-01
+- Added `observer` and `computed` attribute to `@property` decorator.
```

Comment:
Maybe: Added `observer` and `computed` properties to the `@property` decorator options.

Reply:
Done.